repo/packages/C/cassandra/3/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "assets": {
+    "uris": {
+      "scheduler-zip":"https://s3-us-west-2.amazonaws.com/cassandra-framework-dev/ken/scheduler.zip",
```

Comment:
For any EA/GA we are releasing, these should come from a separate directory, to prevent confusion and maintaining quality at all levels. 
So, `s/ken/ea/g` OR `s/ken/ga/g`